packages/docs/docs/dev/reference.mdx

```diff
@@ -194,55 +155,20 @@ The events enum `FuelWalletEvents` can be imported from the `@fuel-wallet/sdk` p
 The `fuel` object has an `events` property which is an enum of all the events that can be listened to. <br />
 The `on` method takes two arguments, the event name and a callback function. The callback function receives data associated with the event.
 
-### Connection
-
-This event is triggered when the connection status change between the Wallet and the application. The callback function receives the `connection` object.
-
-<CodeImport
-  file="../../src/hooks/useIsConnected.tsx"
-  commentBlock="eventAccountChanges"
-/>
-
-### Accounts
-
-This event is triggered when the list of connected accounts to the application change. The callback function receives the `account` string.
-
-<CodeImport
-  file="../../examples/events/Accounts.tsx"
-  commentBlock="eventAccountChanges"
-/>
-
-### Current Account
-
-This event is triggered when the current account is changed in the Fuel Wallet Extension. The callback function receives the `account` address.
-
-This event is only triggered if the application has access to the current account address. Otherwise, it is not triggered.
-
-<CodeImport
-  file="../../examples/events/CurrentAccount.tsx"
-  commentBlock="eventCurrentAccount"
-/>
-
-### Network
-
-This event is triggered when the client connects to a different network. The callback function receives the `network` object.
-
-<CodeImport file="../../examples/events/Network.tsx" commentBlock="network" />
-
-### CurrentConnector
-
-This event is triggered when the Current Connector is updated. The callback function receives the `FuelWalletConnector` object.
-
-<CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventCurrentConnector"
-/>
-
-### Connectors
-
-This event is triggered when the Connectors list is updated. The callback function receives the list of connectors, `Array<FuelWalletConnector>`.
-
-<CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventConnectors"
-/>
+```ts
+fuel.on(fuel.events.connection, (connectionState) => {
+  console.log(connectionState);
+});
+```
+
+### Events
```

Comment:
Is there a better name for this? "Events" under "Events" doesnt really make sense

Reply:
Used "Event Types" for the chart